filebeat/prospector/udp/harvester.go

```diff
@@ -4,11 +4,12 @@ import (
 	"net"
 	"time"
 
-	"github.com/elastic/beats/filebeat/harvester"
-	"github.com/elastic/beats/filebeat/util"
+	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
-	"github.com/elastic/beats/libbeat/publisher/beat"
+
+	"github.com/elastic/beats/filebeat/harvester"
```

Comment:
yes :)

I did find it weird for filebeat having import order filebeat>libbeat and for packetbeat it being the other way around.